index.html

```diff
@@ -2497,6 +2502,13 @@ <h3><dfn>Writing content</dfn></h3>
             </li>
           </ol>
         </li>
+        <li>
+          If |context| is `"smart-poster"` and |records| does not contain a
+          <a>URI record</a> for which the protocol is supported by the UA, or
```

Comment:
Why does the UA need to support the protocol... That makes it impossible for writing an application for just configuring tags for other purposes, like if I write a generic smart poster app so create smart posters and to inspect them

Reply:
fixed